.changeset/few-cows-punch.md

```diff
@@ -0,0 +1,5 @@
+---
+"sit-onyx": patch
```

Comment:
```suggestion
"sit-onyx": minor
```
"patch" is usually bug fixes, "minor" are features and breaking changes are "major"